### PR TITLE
Ubs employee permission 

### PR DESCRIPTION
**Pull request by innayashna**
Ubs employee permission is added for notification controller

### CODE REVIEW

File: core/src/main/java/greencity/controller/ManagementNotificationController.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public class ManagementNotificationController {
         @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
         @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
     })
-    @GetMapping
+    @GetMapping("/get-all")
```
Comment (VadymMakitra) [inline core/src/main/java/greencity/controller/ManagementNotificationController.java]: This path is not correct.

File: core/src/main/java/greencity/controller/ManagementNotificationController.java [hunk 2]
```diff
@@ -56,7 +56,7 @@ public ResponseEntity<PageableDto<NotificationTemplateDto>> getAll(@ApiIgnore Pa
         @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
         @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
     })
-    @PutMapping
+    @PutMapping("/update")
```
Comment (VadymMakitra) [inline core/src/main/java/greencity/controller/ManagementNotificationController.java]: This path is not correct

**Comment by VadymMakitra** [comment]
Please check the rules of building url path here: https://restfulapi.net/resource-naming/

**Review by NataliaHlazova — APPROVED** [review]

**Review by maxnazaruk — COMMENTED** [review]
Looks good for me

**Review by maxnazaruk — APPROVED** [review]

**Review by OlegVat — APPROVED** [review]

**Pull request merged**